core/src/wsv.rs

```diff
@@ -173,6 +173,20 @@ impl<W: WorldTrait> WorldStateView<W> {
             .collect()
     }
 
+    /// Calculates network TPS.
+    ///
+    /// `None` means that transactions can not yet be calculated.
+    #[allow(clippy::float_arithmetic, clippy::cast_precision_loss)]
+    pub fn transactions_per_second(&self) -> Option<f64> {
```

Comment:
```suggestion
    pub fn transactions_per_second(&self) -> Result<f64> {
```
AFAICT, we aren't returning None anywhere in the code. And if we can't compute it, it'd be nice to know why. 

We could also return the bare float, if it's 0, we know something went wrong...

Reply:
Ok, let's return `0`.